ios/SnapchatShare.h

```diff
@@ -0,0 +1,52 @@
+//
+//  GooglePlusShare.h
+//  RNShare
+//
+//  Created by Diseño Uno BBCL on 23-07-16.
+//  Copyright © 2016 Facebook. All rights reserved.
+//
+#import <UIKit/UIKit.h>
+// import RCTConvert
+#if __has_include(<React/RCTConvert.h>)
+#import <React/RCTConvert.h>
+#elif __has_include("RCTConvert.h")
+#import "RCTConvert.h"
+#else
+#import "React/RCTConvert.h"   // Required when used as a Pod in a Swift project
+#endif
```

Comment:
I'm not sure if these are still necessary in 3 different ways? Is it possible to just import directly nowadays?